invoicing.gemspec

```diff
@@ -1,28 +1,36 @@
-$:.push File.expand_path("../lib", __FILE__)
+$LOAD_PATH.push File.expand_path('lib', __dir__)
 
 # Maintain your gem's version:
-require "invoicing/version"
+require 'invoicing/version'
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "invoicing"
+  s.name        = 'invoicing'
   s.version     = Invoicing::VERSION
-  s.authors     = ["Martin Kleppmann"]
-  s.email       = ["@martinkl"]
-  s.homepage    = "http://invoicing.codemancers.com/"
-  s.summary     = "Ruby Invoicing Framework"
+  s.license     = 'MIT'
+  s.authors     = ['Martin Kleppmann', 'Codemancers']
+  s.email       = ['hello@codemancers.com']
+  s.homepage    = 'http://invoicing.codemancers.com/'
+  s.summary     = 'Ruby Invoicing Framework'
   s.description = <<-DESC
-This is a framework for generating and displaying invoices (ideal for commercial
- Rails apps). It allows for flexible business logic; provides tools for tax
- handling, commission calculation etc. It aims to be both developer-friendly
- and accountant-friendly.
-DESC
+    This is a framework for generating and displaying invoices (ideal for commercial
+     Rails apps). It allows for flexible business logic; provides tools for tax
+     handling, commission calculation etc. It aims to be both developer-friendly
+     and accountant-friendly.
+  DESC
 
-  s.files = Dir["{lib}/**/*", "LICENSE", "Rakefile", "README.md"]
+  s.post_install_message = <<-POST_INSTALL
+    1.2.x will be the last version to support Ruby < 2.4 and Rails < 5.
```

Comment:
Should we start the message as a Deprecation Warning: ?

Reply:
That is generally used for classes. Maybe we should add it for FindSubclasses instead?

Reply:
Yes! that works better. Since the plan is to remove the class, it would be better to warn the usage of the class as well 👍